src/main/java/gregtech/api/unification/material/properties/OreProperty.java

```diff
@@ -0,0 +1,138 @@
+package gregtech.api.unification.material.properties;
+
+import gregtech.api.unification.material.Material;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class OreProperty implements IMaterialProperty<OreProperty> {
+
+    /**
+     * List of Ore byproducts.
+     *
+     * Default: none, meaning only this property's Material.
+     */
+    //@ZenProperty
+    private final List<Material> oreByProducts = new ArrayList<>();
+
+    /**
+     * Crushed Ore output amount multiplier during Maceration.
+     *
+     * Default: 1 (no multiplier).
+     */
+    //@ZenProperty
+    private int oreMultiplier;
+
+    /**
+     * Byproducts output amount multiplier during Maceration.
+     *
+     * Default: 1 (no multiplier).
+     */
+    //@ZenProperty
+    private int byProductMultiplier;
+
+    /**
+     * Material to which smelting of this Ore will result.
+     *
+     * Material will have a Dust Property.
+     * Default: none.
+     */
+    //@ZenProperty
+    @Nullable
+    private Material directSmeltResult;
```

Comment:
Why does this exist when there is `smeltInto` on Dust Property?

Reply:
`smeltInto` on the `IngotProperty` essentially only exists for Magnetic Materials

Reply:
ah, I though `smeltInto` was on the Dust Property instead of on the Ingot Property